pkg/acsengine/engine.go

```diff
@@ -564,6 +564,14 @@ func (t *TemplateGenerator) getTemplateFuncMap(properties *api.Properties) map[s
 		"HasWindowsSecrets": func() bool {
 			return properties.WindowsProfile.HasSecrets()
 		},
+		"ExperimentalGpuFlag": func() int {
+			//Only test a single profile, since all agents should be the same size
+			isGpuVM := strings.HasPrefix(properties.AgentPoolProfiles[0].VMSize, "Standard_N")
```

Comment:
what if pool 1 has Standard_N, but pool 0 has Standard_D2_v2?  Will this still enable?